api/src/services/import-services/marking/marking-header-configs.ts

```diff
@@ -140,10 +140,6 @@ export const getMarkingBodyLocationCellValidator = (
   utils: CSVConfigUtils<MarkingCSVStaticHeader>
 ): CSVCellValidator => {
   return (params: CSVParams) => {
-    if (params.cell === undefined) {
-      return [];
-    }
```

Comment:
Should this be based on the `optional` config flag?

Reply:
(Read the doc, nvm!)

Reply:
Yeah I guess the optional vs required cell value is a trickier check. 

For single columns (like taxon or alias) it would be easy enough to have a flag to make them optional. 

But for multi-column concepts, you would have to have some fancier check. Thinking of what it would look like to make "markings" as a concept optional, when it spans multiple columns, some of which are essential for a valid marking, and some which are optional.

Anyways, something for later maybe

Reply:
Yea that's a good point. It reminds me of my brainstorms around row level validations vs cell validations (CSVCellValidator). Currently we only have the ability to validate the individual cells, but I could see some use in being able to create CSVRowValidators for extra validation control around groups or sets of values.

Reply:
You could have it run the row validators before the cell validators to validate structural cell data. 

ie: 
CSVRowValidator: alias + date + time -> maps to a critter capture
CSVCellValidator: alias === 'string' && is in the survey